{{ cookiecutter.repo_name }}/docs/mkdocs/docs/index.md

```diff
@@ -16,4 +17,13 @@ Syncing data to cloud storage
 * `make sync_data_up` will use `gsutil rsync` to recursively sync files in `data/` up to `gs://{{ cookiecutter.dataset_storage.gcs.bucket }}/data/`.
 * `make sync_data_down` will use `gsutil rsync` to recursively sync files in `gs://{{ cookiecutter.dataset_storage.gcs.bucket }}/data/` to `data/`.
 {% endif %}
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+
+## Project layout
+
+    mkdocs.yml    # The configuration file.
+    docs/
+        index.md            # The documentation homepage.
+        getting-started.md  # Set-up instructions
+        ...       # Other markdown pages, images and other files.
```

Comment:
The docs here are for the whole project, not just for the documentation itself, so maybe this doesn't belong here?

Reply:
Great point. Would a `tree` of the high-level default structure be appropriate? We can also just delete.